metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/utils/EnrichmentEntityDao.java

```diff
@@ -92,7 +90,7 @@
   private final MongoClient mongo;
 
   /**
-   * @deprecated see 
+   * @deprecated use {@link #EnrichmentEntityDao(String)}
```

Comment:
My comment was referring to remove the @deprecated since we it's not a deprecation, we can use both and in fact we use similar methods in other connections as well and not a String URI.

Reply:
yes, I understood the meaning. But I still think that this constructor shouldn't be used anymore and should be removed, as any code invoking this constructor will not work in production environments.